pypeit/images/rawimage.py

```diff
@@ -195,42 +193,35 @@ def process(self, process_steps, par, bpm=bpm, flatimages=None, bias=None,
         self.par = par
         self._bpm = bpm
 
-        # For error checking
-        steps_copy = process_steps.copy()
         # Get started
         # Standard order
         #   -- May need to allow for other order some day..
-        if 'subtract_overscan' in process_steps:
+        if par['use_overscan']:
```

Comment:
I'm pretty sure that we shouldn't be subtracting the overscan and the bias... perhaps this should be:

```
if par['use_overscan']:
    self.subtract_overscan()
elif par['use_biasimage']:
    self.subtract_bias(bias)
```

on that note, I presume the overscan is subtracted from the dark, right? So that the master dark *only* contains the dark counts?

Reply:
Hi @rcooke-ast, the bias should be overscan subtracted. Biases are meant to subtract out the part of the bias that is spatially coherent but not time dependent. In other words, overscan takes out the DC offset, bias removes the remaining fixed pattern. If you don't overscan subtract the bias then if the DC offset varies the biases become meaningless.

Reply:
Sounds good to me - in that case, we should set `use_overscan=True` in `pypeitpar.CalibrationsPar`. The default for `biasframe` is currently False.